AnalyticsData/owlbot.py

```diff
@@ -34,10 +36,19 @@
     src=src,
     dest=dest,
     copy_excludes=[
-        src / "**/*_*.php"
+        src / "**/[A-Z]*_*.php"
```

Comment:
Hmm, is `beta_analytics_data_rest_client_config.php` included in these lists? If so, we definitely need to update all of these 😞 

Would it be better to submit a PR to update all of these?

Reply:
Added [here](https://github.com/googleapis/google-cloud-php/pull/4987)

Would it also be better to be specific by directory instead of using the double asterisk? I think in this case we would want `src / "*/[A-Z]*_*.php"`